index.html

```diff
@@ -11,6 +11,56 @@
 </head>
 
 <body>
+    <div class="popup-window-container hide">
+        <div class="popup-window">
+          <div class="project-heading">
+            <h2 class="popup_title"></h2>
```

Comment:
- [ ] Nice work 👍 , Kindly Fix your heading in the desktop view
    how your design looks 👇 
      
![image](https://user-images.githubusercontent.com/66097371/162068996-f873cd9e-dc46-4288-ac3f-800e7d3fc13b.png)